From b53d207faf9c5a9b3383139c401144bca86535d3 Mon Sep 17 00:00:00 2001
From: Ilya Ledvich <ilya@compulab.co.il>
Date: Thu, 2 Jan 2020 17:32:33 +0200
Subject: [PATCH 18/39] ARM: dts: imx7d: cl-som-imx7: disable unsupported
 operating point

Disable 'opp-1200000000' node in the 'cpu0_opp_table' as it's not supported by the CPU
variant used in Compulab devices based on i.MX7 SoC.

Signed-off-by: Ilya Ledvich <ilya@compulab.co.il>
---
 arch/arm/boot/dts/imx7d-cl-som-imx7.dts | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
index 9864c508cb9e..ca76d8fdbf3b 100644
--- a/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
+++ b/arch/arm/boot/dts/imx7d-cl-som-imx7.dts
@@ -123,6 +123,12 @@
 	cpu-supply = <&sw1a_reg>;
 };
 
+&cpu0_opp_table {
+	opp-1200000000 {
+		status = "disabled";
+	};
+};
+
 &ecspi1 {
 	fsl,spi-num-chipselects = <2>;
 	pinctrl-names = "default";
-- 
2.20.1

